### ISSUE TITLE
[Classic 1.13.3] Can't use items from bags.

### ISSUE THREAD

**Issue by Ghamarro** [tullamods/Bagnon#1046]
For some reason everytime I try to use an item from my bags by left clicking on it, I get a prompt that says the action was blocked by an addon and it asks if I would like to disable it, providing me with a quick disable button on the prompt window, which if I press, it disables Bagnon, and then everything works fine again. Also happens with me trying to use disenchanting on items in bags. I can drag items to a hotbars and use them from there, though.

**Comment by InTheMorning** [comment]
can confirm this is happening for me as well.
```
Message: Error: AddOn Bagnon attempted to call a forbidden function (UNKNOWN()) from a tainted execution path.
Debug:
   [C]: ?
   [C]: UseContainerItem()
   ..\FrameXML\ContainerFrame.lua:1254: ContainerFrameItemButton_OnClick()
   [string "*:OnClick"]:12:
      [string "*:OnClick"]:1
   [C]: ?
```

**Comment by Mexallon** [comment]
I got the same issue. Can't even disenchant items. This addon is unusable in the current bugged version.

**Comment by leonardopsantos** [comment]
Same here, had to disable Bagnon :-(

**Comment by leonardopsantos** [comment]
How is this a duplicate? The second newest issue is from October. Bagnon worked perfectly in 1.13.2. Which issue this is issue is a duplicate?

**Comment by leonardopsantos** [comment]
Seems to be fixed in v8.2.21
